scanpy/plotting/_tools/scatterplots.py

```diff
@@ -893,7 +896,7 @@ def _get_data_points(
                 tuple(number - 1 for number in comp) for comp in components_list
             ]
     else:
-        data_points = [adata.obsm[basis_key][:, offset : offset + n_dims]]
+        data_points = [np.array(adata.obsm[basis_key])[:, offset : offset + n_dims]]
```

Comment:
Just wanted to point out that this is needed otherwise the flipping would change at every call of the function (meaning it would be flipped forward and reverse at every call)